src/Nancy.ViewEngines.DotLiquid/project.json

```diff
@@ -16,10 +16,15 @@
 
     "dependencies": {
         "Nancy": { "target": "project" },
-        "DotLiquid": "1.7.0"
+        "DotLiquid": "2.0.55"
     },
 
     "frameworks": {
+	    "netstandard1.6": {
+            "imports": [
+                "portable-net451+win8"

```

Comment:
Is this import really needed? It supports `netstandard1.3` in the package
![image](https://cloud.githubusercontent.com/assets/50543/23836276/094446ba-0776-11e7-9b1b-c344cdd2fecc.png)